core/templates/dev/head/pages/exploration_editor/editor_tab/customize_interaction_modal_directive.html

```diff
@@ -29,10 +29,20 @@ <h3 ng-if="stateInteractionIdService.displayed && (customizationModalReopened ||
 
   <form ng-if="stateInteractionIdService.displayed && hasCustomizationArgs" name="form.schemaForm" class="protractor-test-interaction-editor">
     <div ng-repeat="customizationArgSpec in customizationArgSpecs track by $index">
+      <div ng-if="customizationArgSpec.schema.type !== 'bool'">
       <div class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
-      <div>
-        <schema-based-editor local-value="stateCustomizationArgsService.displayed[customizationArgSpec.name].value" schema="customizationArgSpec.schema">
-        </schema-based-editor>
+        <div>
+          <schema-based-editor local-value="stateCustomizationArgsService.displayed[customizationArgSpec.name].value" schema="customizationArgSpec.schema">
+          </schema-based-editor>
+          <div style="margin-bottom: 15px;"></div>
```

Comment:
If this is used in both branches would it be possible to take it out of the ng-ifs and put it below both? The less code that's branched on, the better -- it removes duplication.

Bonus points if you can add a CSS class to the schema-based-editor tag and do it there instead (see below).